packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1908,16 +1913,22 @@ export class MergeTree {
 			segmentGroup: undefined,
 		};
 
-		const { segment: startSeg } = this.getContainingSegment(start, refSeq, clientId);
-		const { segment: endSeg } = this.getContainingSegment(end - 1, refSeq, clientId);
+		const { segment: startSeg } =
+			start.pos === -1
```

Comment:
should this ever happen? i don't see any tests for it.

Reply:
found a few tests. still not sure we want to introduce this yet, as it seems like scope creep.